README.md

```diff
@@ -71,17 +71,18 @@ yarn storybook:axe
-| `--pattern`           | `.*`               | regex pattern                                                                               | Only run tests that match a component name pattern                                                                                                                          |
-| `--reporter`          | `spec`             | spec, dot, nyan, tap, landing, list, progress, json, json-stream, min, doc, markdown, xunit | How to display the test run. Can be any [built-in Mocha reporter](https://mochajs.org/#reporters).                                                                          |
-| `--reporter-options`  |                    | string                                                                                      | Options to pass to the mocha reporter. Especially useful with the xunit reporter - e.g. `--reporter-options output=./filename.xml`                                          |
-| `--storybook-address` |                    | url                                                                                         | Storybook server address to test against instead of using a static build directory. If set, `--build-dir` will be ignored. e.g. `--storybook-address http://localhost:6006` |
-| `--timeout`           | 2000               | number                                                                                      | Timeout (in milliseconds) for each test                                                                                                                                     |
+| Option | Default | Values | Description |
+| ------ | ------- | ------ | ----------- |
+| `--browser` | `chromium` | chromium, firefox, webkit | Which browser to run the tests |
+| `--build-dir` | `storybook-static` | path | Storybook static build directory |
+| `--failing-impact` | `all` | all, minor, moderate, serious, critical | The lowest impact level that should be considered a failure |
+| `--headless` | `true` | boolean | Whether to run headlessly or not |
+| `--pattern` | `.*` | regex pattern | Only run tests that match a component name pattern |
+| `--reporter` | `spec` | spec, dot, nyan, tap, landing, list, progress, json, json-stream, min, doc, markdown, xunit | How to display the test run. Can be any [built-in Mocha reporter](https://mochajs.org/#reporters). |
+| `--reporter-options` |  | string | Options to pass to the mocha reporter. Especially useful with the xunit reporter - e.g. `--reporter-options output=./filename.xml` |
+| `--storybook-address`|  | url | **_Deprecated!_** Use `--storybook-url` instead. |
+| `--storybook-url` |  | url | Url to a running Storybook to test against. Alternative to `--build-dir`, which will be ignored if this is set. |
+| `--timeout` | 2000 | number | **_Deprecated!_** Use the `timeout` story parameter instead. |
```

Comment:
Made the table formatting more reasonable to edit while I was here.

Also added some deprecation messages to the affected options.